articles/howto-build-astatic-site-with-gatsbyjs/index.md

```diff
@@ -0,0 +1,281 @@
+
+Congratulations! You just built your first static site using GatsbyJS.
+
+
+Now let’s talk about your file directory, it looks overwhelming especially if you are still a beginner using [Node.js](https://nodejs.org/en/).
+
+-	**The node-modules folder:** Contains a bunch of files you may or may not need.
+-	**The public folder:** This contains all of your finished static sites that would be served on a cloud server.
+-	**The src folder:** This is the folder you will be using the most because this is where you are going to store the pages you are building for your site.
+You will find an  ***index.js*** file in the pages folder because Gatsby uses *Node.js* and *React.js* and these files will be a JavaScript file but you can also use a markdown file as well with the file extension of '.md'.
+-	**The gitignore file:** This file tells GitHub to ignore whatever file you specify in it ignore. For example API keys.
+-	**The package.json:** Contain the dependencies or packages you have installed using the node package manager *“npm install”*
+
+### Adding content to the site
+
+As excited as I am that you have created your first GatsbyJS static site, let's move forward by adding more content to the site by making it look better.
+Remember how I mentioned you will need some knowledge of React.js?
+
+Yes, good! The moment has come for us to use it now, but don’t worry we will just be working with the basics here that are pretty easy.
+
+In the index.js file add this piece of code:
+
+```JavaScript
+// index.js code
+import React from "react"
+
+export default function Home() {
+return <div style={{ color: 'tomato' }}>
+    <h1>Hello world!</h1>
+    // Your new paragraph or content
+    <p>Welcome to my first Gatsby site</p>
+</div>
+}
+```
+
+### Linking between pages
+
+Having a one-page static site doesn’t sound cool at all, what if you want to have 2 or more pages? To do this create another page and link them together, so that you can easily navigate through the added pages.
+
+First of all, import a Gatsby react link and add it to the top of the index.js file
+`import { Link } from "gatsby"`.
+
+Secondly, create a .js file with the name **page-2.js** in the pages folder and add this code snippet:
+
+```JavaScript
+//page-2 code
+import React from "react"
+import { Link } from "gatsby"
+
+export default function Home() {
+    return <div style={{ color: 'tomato' }}>
+        <h1>Welcome to Page 2</h1>
+        // To go back to the homepage
+        <Link to="/">Back</Link>
+    </div>
+}
+```
+
+Then finally link it using the Gatsby link by placing this code in the index.js file to connect it to the page-2.js file
+
+```bash
+<Link to=”/page-2/”>Page 2</Link>
+```
+
+### Counter.js file
+
+Here you are going to make the site more interactive by adding a counter to the site where you can click on a **plus** button to increase a number and a **minus** button to decrease a number.
```

Comment:
- We will now make the site more...
- Missing comma before "where"

Reply:
Done.